README.md

```diff
@@ -1,68 +1,125 @@
+change its state.
+
+State is used to handle several things in your component:
+
+- Interactivity (e.g. changing data when a user clicks a button)
+- Fetching remote data (remote data is, by definition, not available right away when the component is mounted — state
+gives us a way of updating the component once that data arrives)
+- Reacting to the passing of time (i.e. setting an interval or timeout)
+
+## Props vs state
+As mentioned before, it's important to know the difference between props and state. Props and state are used as input
+for the `render()` method to determine its output, but they are _not_ the same thing! The best way to figure out if data
+should go in props or state is to ask ourselves _'Will this data ever change?'_. If not, it's a prop. If it will, it
+should go in state! Keep in mind that whenever props _and/or_ state change, the component will run its `render()` method
+again.
+
+## Setting initial state
+![An old radio thing](http://support.radioshack.com/support_tutorials/communications/Images/pro2067e_init.gif)
+
+Enough talk, let's see some more code! In this lesson, we'll focus on just setting the _initial state_. Since we're just
+setting the initial state, it will remind you very much of props (since the data won't change). Keep in mind, though,
+that we're able to _change_ this state whenever we want, making for a very powerful feature for dynamic components.
+
+Let's say we have a `<ToggleButton />` component. A toggle button has an on and off state. Using props for that value
+wouldn't work, since we can't actually change our props! Let's use state for this instead. We'll assume that the
+default state of this component is to be in the _off_ state. Let's call that state property `isEnabled`. Setting the
+initial state is done in the `constructor` of our ES2015 class component:
 
-Similarly to how we taught props, we want to walk students through the
-difference between `getInitialState()` and setting `this.state` in the
-constructor.
+```js
+class ToggleButton extends React.Component {
+  constructor() {
+    super();
+    
+    this.state = {
+      isEnabled: false
+    };
+  }
 
-So for example, we'd have
+  render() {
+    return (
+      <div className="toggle-button">I am toggled {this.state.isEnabled ? 'on' : 'off'}.</div>
+    );
+  }
+}
+```
 
-```javascript
-const Toggler = React.createClass({
+If, for some reason, we were still using `React.createClass()`, we would add a method called `getInitialState()` that
+returns our state object:
+
+```js
+const ToggleButton = React.createClass({
   getInitialState() {
     return {
-      on: false
-    }
+      isEnabled: false
+    };
   },
 
   render() {
     return (
-      <div>
-       {React.cloneElement(
-         this.props.children,
-         { ...c.props, on: this.state.on })}
-      </div>
-    )
+      <div className="toggle-button">I am toggled {this.state.isEnabled ? 'on' : 'off'}.</div>
+    );
   }
-})
+});
```

Comment:
Good idea to show them the old method as well, as there's a good chance they'll run into that.